release_notes/373-release-notes.adoc

```diff
@@ -15,22 +15,73 @@ toc::[]
-
-{product-title-short} 3.73 includes:
-
-
 [id="new-features-373"]
 == New features
 
-[id="doc-updates-373"]
-=== Documentation updates
-
+[id="rhacs-cloud-service"]
+=== Red Hat Advanced Cluster Security Cloud Service
+
+[IMPORTANT]
+====
+{product-title-short} Cloud Service is available as a Field Trial release.
+A Field Trial provides approved customers access to a release with general availability-level quality and limited support.
+For more information about accessing {product-title-short} Cloud Service, contact link:https://www.redhat.com/en/contact[Red Hat Sales].
+====
+
+Red Hat Advanced Cluster Security Cloud  Service is a fully managed software-as-a-service to protect containerized applications and orchestration platforms across the entire application lifecycle. Advanced Cluster Security Cloud Service helps shift the operational and management responsibility and support to Red Hat.
+
+{product-title-short} Cloud Service extends beyond Red Hat OpenShift and includes Kubernetes services from all major cloud providers such as Amazon Elastic Kubernetes Service (EKS), Microsoft Azure Kubernetes Service (AKS), and Google Kubernetes Engine (GKE).
+
+For more information and to try {product-title-short} Cloud Service, see link:https://www.redhat.com/en/engage/security-managed-service-20221011[Request early access to Red Hat Advanced Cluster Security Cloud Service].
+
+[NOTE]
+====
+The following documentation sections does not apply to {product-title-short} Cloud Service:
+
+* xref:../installing/install-ocp-operator.adoc#install-central-operator_install-ocp-operator[Installing Central]
+* xref:../installing/installing_helm/install-helm-quick.adoc#installing-quickly_acs-install-helm-quick[Installing the central-services Helm chart without customization]
+* xref:../installing/installing_helm/install-helm-customization.adoc#install-central-services-helm-chart_acs-install-helm-customization[Installing the central-services Helm chart]
+* xref:../upgrading/upgrade-helm.adoc#upgrading-using-helm-charts[Upgrading using Helm charts]
+* xref:../upgrading/upgrade-roxctl.adoc[Upgrading the Central cluster
+]
+* xref:../configuration/enable-offline-mode.adoc[Enabling offline mode]
+* xref:../configuration/expose-portal-over-http.adoc[Exposing the {product-title-short} portal over HTTP]
+* xref:../configuration/configure-proxy.adoc[Configuring a proxy for external network access]
+* xref:../configuration/configure-endpoints.adoc[Configuring endpoints]
+* xref:../configuration/monitor-acs.adoc[Monitoring with Prometheus]
+* xref:../operating/manage-user-access/configure-ocp-oauth.adoc[Configuring OpenShift Container Platform OAuth server as an identity provider]
+* xref:../backup_and_restore/backing-up-acs.adoc[Backing up Red Hat Advanced Cluster Security for Kubernetes]
+* xref:../backup_and_restore/restore-acs.adoc[Restoring from a backup]
+====
+
+[id="vulnerability-dashboard-changes"]
+==== Vulnerability dashboard changes for Cloud Service users
```

Comment:
I would reword this. Maybe: `Cloud Service Vulnerability Changes Dashboard`

Reply:
There is a  "Vulnerability dashboard" which now has some changes when people use Cloud Service. That suggestion would change its meaning, so I'm keeping it the same.